app-template/Cargo.toml

```
@@ -37,23 +37,23 @@ cosmwasm-std = { version = "1.2" }
 cosmwasm-schema = { version = "1.2" }
 cw-controllers = { version = "1.0.1" }
 cw-storage-plus = "1.1.0"
-thiserror = { version = "<=1.0.48" }
+thiserror = { version = "1.0.49" }
```

Glad to see `thiserror` finally fixed the bug!
It would use the `1.0.50` version
```suggestion
thiserror = { version = "1.0.50" }
```